jwst/assign_wcs/util.py

```diff
@@ -259,13 +259,8 @@ def wcs_from_footprints(dmodels, refmodel=None, transform=None, bounding_box=Non
 
     out_frame = refmodel.meta.wcs.output_frame
     input_frame = dmodels[0].meta.wcs.input_frame
-    wnew = wcs_from_fiducial(fiducial, coordinate_frame=out_frame, projection=prj, transform=transform)
-    # temporary fix before gwcs 0.14 is released
-    # wnew = wcs_from_fiducial(fiducial, coordinate_frame=out_frame, projection=prj,
-    #                          transform=transform, input_frame=input_frame)
-    pipe = wnew.pipeline[:]
-    pipe[0] = (input_frame, pipe[0][1])
-    wnew = WCS(pipe)
+    wnew = wcs_from_fiducial(fiducial, coordinate_frame=out_frame, projection=prj,
+                             transform=transform, input_frame=input_frame)
```

Comment:
Yes, the fix in gwcs is released now - good you remembered 👍

Reply:
The deprecation warning you put into `gwcs` helped.  I was seeing:

```
/Users/jdavies/miniconda3/envs/jwst/lib/python3.8/site-packages/gwcs/wcs.py:973: DeprecationWarning: Indexing a WCS.pipeline step is deprecated. Use the `frame` and `transform` attributes instead.
    warnings.warn("Indexing a WCS.pipeline step is deprecated. "
```